.clang-format

```diff
@@ -43,7 +43,6 @@ BraceWrapping:
   SplitEmptyNamespace: false
 BreakBeforeBinaryOperators: None
 BreakBeforeBraces: Custom
-BreakBeforeConceptDeclarations: Always
```

Comment:
There already are unsupported `clang-format` option in this file which are commented out (e.g. `InsertTrailingCommas`), to maintain consistency, either the options incompatible with Ubuntu should be commented out, or all the incompatible options should be deleted. Do as you want, I've no opinion, but it should be consistent.